packages/functional-tests/tests/react-conversion/oauthSignup.spec.ts

```diff
@@ -97,49 +100,23 @@ test.describe('severity-1 #smoke', () => {
       await relier.signOut();
     });
 
-    test('signup oauth webchannel - sync mobile or FF desktop 123+', async ({
+    test('signup oauth webchannel with Sync - FF desktop 123+', async ({
       target,
-      syncBrowserPages: { confirmSignupCode, page, login, signup },
+      syncOAuthBrowserPages: { confirmSignupCode, page, login, signup },
       testAccountTracker,
     }) => {
-      test.fixme(true, 'Fix required as of 2024/06/28 (see FXA-10003).');
       const { email, password } =
         testAccountTracker.generateSignupAccountDetails();
-      const customEventDetail: FxAStatusResponse = {
-        id: 'account_updates',
-        message: {
-          command: FirefoxCommand.FxAStatus,
-          data: {
-            signedInUser: null,
-            clientId: FF_OAUTH_CLIENT_ID,
-            capabilities: {
-              pairing: false,
-              multiService: false,
-              choose_what_to_sync: true,
-              engines: ['bookmarks', 'history'],
-            },
-          },
-        },
-      };
-
-      await signup.goto('/authorization', syncMobileOAuthQueryParams);
+
+      await signup.goto('/authorization', syncDesktopOAuthQueryParams);
 
       await signup.fillOutEmailForm(email);
 
       await expect(signup.signupFormHeading).toBeVisible();
 
-      await signup.sendWebChannelMessage(customEventDetail);
-
-      // Only engines provided via web channel for Sync mobile are displayed
       await expect(signup.CWTSEngineHeader).toBeVisible();
       await expect(signup.CWTSEngineBookmarks).toBeVisible();
       await expect(signup.CWTSEngineHistory).toBeVisible();
-      await expect(signup.CWTSEnginePasswords).toBeHidden();
-      await expect(signup.CWTSEngineAddons).toBeHidden();
-      await expect(signup.CWTSEngineOpenTabs).toBeHidden();
-      await expect(signup.CWTSEnginePreferences).toBeHidden();
-      await expect(signup.CWTSEngineCreditCards).toBeHidden();
```

Comment:
Why remove these / the comment? 👀

Reply:
Yea, this was an unexpected result. The desktop oauth flow returns the sync engines and not the mocked engines. The way we mock webchannels does not seem to work the same when oauth is enabled. This tests asserts the engines from the browser.